src/widgets/Tabs/VaccineSelect.js

```diff
@@ -136,46 +168,70 @@ const VaccineSelectComponent = ({
 };
 
 const mapDispatchToProps = dispatch => {
+  const onRefuse = value => dispatch(VaccinePrescriptionActions.refuse(value));
   const onCancelPrescription = () => dispatch(VaccinePrescriptionActions.cancel());
   const onSelectBatch = itemBatch => dispatch(VaccinePrescriptionActions.selectBatch(itemBatch));
   const onSelectVaccine = vaccine => dispatch(VaccinePrescriptionActions.selectVaccine(vaccine));
+  const onSelectVaccinator = vaccinator =>
+    dispatch(VaccinePrescriptionActions.selectVaccinator(vaccinator));
   const onConfirm = () =>
     batch(() => {
       dispatch(VaccinePrescriptionActions.confirm());
       dispatch(goBack());
     });
 
-  return { onCancelPrescription, onConfirm, onSelectBatch, onSelectVaccine };
+  return {
+    onCancelPrescription,
+    onConfirm,
+    onRefuse,
+    onSelectBatch,
+    onSelectVaccine,
+    onSelectVaccinator,
+  };
 };
 
 const mapStateToProps = state => {
+  const hasRefused = selectHasRefused(state);
   const selectedRows = selectSelectedRows(state);
   const selectedBatchRows = selectSelectedBatchRows(state);
   const selectedBatches = selectSelectedBatches(state);
   const selectedVaccines = selectSelectedVaccines(state);
   const vaccines = selectVaccines(state);
   const [selectedVaccine] = selectedVaccines;
 
-  return { selectedBatches, selectedBatchRows, selectedRows, selectedVaccine, vaccines };
+  return {
+    vaccinator,
```

Comment:
the `vaccinator` isn't set in the method - is there a commit missing?

Reply:
Ah yeah I think there is a commit missing which will mess the next PR up also 😡

Reply:
f8a8e47

sorry!